meta-ostro/conf/local.conf.sample

```diff
@@ -2,11 +2,11 @@
 # This file is your local configuration file and is where all local user settings
 # are placed. The comments in this file give some guide to the options a new user
 # to the system might want to change but pretty much any configuration option can
-# be set in this file. More adventurous users can look at local.conf.extended 
+# be set in this file. More adventurous users can look at local.conf.extended
```

Comment:
Seems like we don't have it at all in ostro-os. Might be a Yocto thing. I can remove the entire sentence.